autohooks/install.py

```diff
@@ -17,13 +17,17 @@
 
 import shutil
 
-from setuptools.command.install import install
 from setuptools.command.develop import develop
+from setuptools.command.install import install
 
+from autohooks.config import load_config_from_pyproject_toml
 from autohooks.utils import (
     get_git_hook_directory_path,
     get_autohooks_directory_path,
-)
+    get_pyproject_toml_path)
+
+pyproject_toml = get_pyproject_toml_path()
+config = load_config_from_pyproject_toml(pyproject_toml)
```

Comment:
Could you move the pyproject_toml and config variables down into the get_pre_commit_hook_template_path function? Module variables can be imported from other modules and both variables are only used in the get_pre_commit_hook_template_path function.